index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>New York Times</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">	
+</head>
+<body>
+	<header>
+		<ul>
+			<li><i class = "fa fa-bars"></i></li>
+			<li><i class = "fa fa-search"></i></li>
+			<li><b>SPACE & COSMOS</b></li>
```

Comment:
As a plus, you could make the font [sans-serif](https://www.webopedia.com/TERM/S/sans_serif.html)